### DISCUSSION TITLE
Launch in Binder/Collab icon/button

### DISCUSSION THREAD

**Discussion by jpfeuffer** [pradyunsg/furo#513] [Ideas] [answered]
Hi!
Thanks for the beautiful theme. It is by far our favorite.
However, one feature is missing for us to make it perfect:

A launch on Binder or Google Colab button. Could be just a little 🚀 icon next to the dark/light switch on both mobile and desktop.

I saw https://github.com/pradyunsg/furo/issues/57 but we don't need a large customizable top bar. Just an additional icon or two.

We might even have a student that may take a look at it. What do you think?

**Comment by jpfeuffer** [comment]
I guess it would boil down to making top_of_page_buttons a list and allowing another component that users can choose from.
https://github.com/pradyunsg/furo/blob/7933eb886b13890342cfa7e9a0030bd8f034fdf0/src/furo/theme/furo/page.html#L71

**Comment by pradyunsg** [comment, marked as answer]
That's reasonable, although I don't want to implement this as a "collab/binder" button. I'd actually prefer to (as you suggest) make this a list and allowing "custom" as an option within that, allowing for a user defined template to be used in this. Figuring out some way to migrate/handle the list vs non-list variants of that options is a problem for future me though. :)

Would you be willing to file a [trackable change issue](https://github.com/pradyunsg/furo/issues/new?template=issue.yml) for making the top of page button more configurable as a list with `custom` being an option? (see https://github.com/pradyunsg/furo/issues/478 for an example of what those look like)
